create-module.sh

```diff
@@ -136,19 +153,39 @@ EOF
 # Create README.md
 readme="$reactPath/README.md"
 echo -e "Creating ${CYAN}$readme${NC}"
+if [ $stable ] ; then
 cat > $readme << EOF
 # Canvas Kit $upperName
 EOF
+else
+cat > $readme << EOF
+# Canvas Kit $upperName
+
+<a href="https://github.com/Workday/canvas-kit/tree/master/modules/_labs/README.md">
+  <img src="https://img.shields.io/badge/UNSTABLE-alpha-orange" alt="UNSTABLE: Alpha" />
+</a>  This component is work in progress and currently in pre-release.
+EOF
+fi
 
 # Create tsconfig.json
 tsconfig="$reactPath/tsconfig.json"
 echo -e "Creating ${CYAN}$tsconfig${NC}"
+
+if [ $stable ] ; then
```

Comment:
My eyes may be deceiving me but the if/else blocks are the same right? Is this just a preemptive measure in case we need separate logic?

Reply:
Ah - good catch. It's left over from before I pulled out the `rootPath`